content/guides/references/trade-offs.md

```diff
@@ -268,3 +270,139 @@ app.listen(8081, () => {})
 This avoids ever needing a second open browser, but still gives you an
 end-to-end test that provides 100% confidence that the two clients can
 communicate with each other.
+
+### Navigation Rules
+
+By itself, Cypress limits each test to only visiting domains that are considered
+to be same superdomain to one another. If a navigation occurs that does not meet
+the same superdomain rule, the [`cy.origin`](/api/commands/origin) command must
+be used to execute assertions inside the newly navigated origin.
+
+But what is same superdomain? It is actually very similar to that of same
+origin! Two URLs have the same origin if the protocol, port (if specified), and
+host are the same for both. Cypress automatically handles hosts of the same
+superdomain by injecting the
+[`document.domain`](https://developer.mozilla.org/en-US/docs/Web/API/Document/domain)
+property into the visited `text/html` pages. This is why navigations without the
+use of the [`cy.origin`](/api/commands/origin) command are solely scope to same
+superdomain.
+
+Given the URLs below, all have the same superdomain compared to
+`https://www.cypress.io`.
+
+- `https://cypress.io`
+- `https://docs.cypress.io`
+- `https://example.cypress.io/commands/querying`
+
+The URLs below, however, will have different superdomains/origins compared to
+`https://www.cypress.io`.
+
+- `http://www.cypress.io` (Different protocol)
+- `https://docs.cypress.io:81` (Different port)
+- `https://www.auth0.com/` (Different host of different superdomain)
+
+The `http://localhost` URLs differ if their ports are different. For example,
+the `http://localhost:3000` URL is considered to be a different origin from the
+`http://localhost:8080` URL.
+
+The rules are:
+
+- <Icon name="exclamation-triangle" color="red"></Icon> You **cannot**
+  [visit](/api/commands/visit) two domains of different superdomains in the same
+  test and continue to run assertions without the use of the
+  [`cy.origin`](/api/commands/origin) command.
+- <Icon name="check-circle" color="green"></Icon> You **can**
+  [visit](/api/commands/visit) two or more domains of different origin in
+  **different** tests without needing [`cy.origin`](/api/commands/origin).
+
+We understand this is a bit complicated to understand, so we have built a nifty
+chart to help clarify the differences!
+
+#### Parts of a URL
+
+```
```

Comment:
@chrisbreiding added the parts of a url chart here that we can link through our other docs. This area felt appropriate but lmk what you think.